config.rb

```diff
@@ -1,187 +1,3 @@
-  text_str = text_str.gsub(ERB::Util.url_encode(TMPL_CLOSE_TAG),
-                           EMAIL_PRV_CLOSE_TAG)
-  text_str = text_str.gsub('( https://www.facebook.com/studentbeans )', '')
-  text_str = text_str.gsub('( https://twitter.com/studentbeans )', '')
-  text_str
-end
-
-def directory_builder
-  html = '<html><body><ul>'
-  Dir.glob(build_dir + '/emails/*/').each do |folder|
-    html += '<li><a href='
-    # {folder.sub('build/', '')}'>#{folder.sub('build/emails', '')}</a></li>'
-    # sub
-    subhtml = '<html><body><ul>'
-    Dir.glob(folder + '/*.html').each do |file|
-      # relPath = file.split('/')[-1]
-      subhtml += '<li><a href=' # {relPath}'>#{relPath}</a></li>'
-    end
-    subhtml += '</ul></body></html>'
-    puts folder + 'index.html'
-    File.open(folder + 'index.html', 'w+') do |content|
-      content.puts(subhtml)
-    end
-    # /sub
-  end
-  html += '</ul></body></html>'
-  File.open('build/index.html', 'w+') do |content|
-    content.puts(html)
-  end
-end
-
-class InlineCSS < Middleman::Extension
-  def initialize(app, options_hash = {}, &block)
-    super
-    app.after_build do |builder|
-      Dir.glob(build_dir + File::SEPARATOR + '**/*.html').each do |source_file|
-        # if source_file.start_with? 'build/partials'
-        # premailer = Premailer.new(source_file,
-        # verbose: true, css: 'http://localhost:4567/stylesheets/all.css',
-        # remove_classes: false,
-        # adapter: 'nokogiri')
-        # else
-        premailer = Premailer.new(
-          source_file,
-          verbose: true,
-          remove_classes: false,
-          adapter: 'nokogiri'
-        )
-        # end
-        destination_file = source_file.gsub('.html', '--inline.html')
-        destination_txt_file = source_file.gsub('.html', '.txt')
-
-        puts "Inlining file: #{source_file} to #{destination_file}"
-
-        # unless source_file.start_with? 'build/partials'
-        File.open(destination_txt_file, 'w') do |content|
-          content.puts(parse_text(premailer.to_plain_text))
-        end
-
-        File.open(destination_file, 'w') do |content|
-          content.puts(parse(premailer.to_inline_css))
-        end
-        # end
-
-        File.delete(Dir.getwd + File::SEPARATOR + source_file)
-      end
-
-      directory_builder
-    end
-  end
-end
-::Middleman::Extensions.register(:inline_css, InlineCSS)
-
-# ====================================================
-# Build
-# ====================================================
-
-configure :build do
-  activate :i18n, path: 'emails/:locale/', mount_at_root: false
-  activate :inline_css
-  # Enable cache buster
-  activate :asset_hash
-  # Use relative URLs
-  activate :relative_assets
-end
-
-# ====================================================
-# Deploy
-# ====================================================
-
-activate :deploy do |deploy|
-  deploy.method = :git
-end
+preferred_syntax = :scss
+sass_dir = './source/stylesheets'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Useless assignment to variable - `sass_dir`.](https://www.codacy.com/app/vincent_4/email_transactional/file/2005412829/issues/source?bid=3123097&fileBranchId=3123101#l2)